src/js/components/DataTable/Header.js

```diff
@@ -66,14 +66,17 @@ const Header = ({
               typeof header === 'string' ? <Text>{header}</Text> : header;
 
             if (onSort && sortable !== false) {
-              const Icon =
-                onSort &&
-                sortable !== false &&
-                sort &&
-                sort.property === property &&
-                theme.dataTable.icons[
-                  sort.direction !== 'asc' ? 'ascending' : 'descending'
-                ];
+              let Icon;
+              if (onSort && sortable !== false) {
```

Comment:
I think the icon should still work if `sortable` isn't false and `onSort` wasn't defined.

Reply:
The point of the icon is to indicate that the user could sort on that column. We don't want to show the icon if the user cannot sort on the column.

Reply:
I think my point is that one can use `sortable` prop without `onSort` and still expect the new icon functionality to work as expected. For example, this story https://storybook.grommet.io/?path=/story/datatable--controlled-grouped
uses only `sortable` prop, without `onSort` and I can see a use case of it using the new sortable icon.

Reply:
Thank you for clarifying.